src/Rector/AbstractPHPUnitRector.php

```diff
@@ -0,0 +1,17 @@
+<?php declare(strict_types=1);
+
+namespace Rector\Rector;
+
+use Nette\Utils\Strings;
+use PhpParser\Node;
+use Rector\Node\Attribute;
+
+abstract class AbstractPHPUnitRector extends AbstractRector
+{
+    protected function isInTestClass(Node $node): bool
+    {
+        $className = $node->getAttribute(Attribute::CLASS_NAME);
+
+        return Strings::endsWith((string) $className, 'Test');
```

Comment:
Could you add comment there, that this is only approximate assumption?
It might be important in some edge cases, where these Rectorswould fail, so it's easier to debug.

Reply:
Also the note that from PR would be great: https://phpunit.de/getting-started-with-phpunit.html